src/handler/http/models/dashboards.rs

```diff
@@ -43,6 +44,13 @@ pub struct ListDashboardsQuery {
     /// The optional case-insensitive title substring with which to filter
     /// dashboards.
     title: Option<String>,
+
+    /// The optional number of dashboards to retrieve. If not set then all
+    /// dashboards that match the query parameters will be returned.
+    ///
+    /// Currently this parameter is only untilized by the API when the `title`
+    /// parameter is also set.
+    page_size: Option<u64>,
 }
```

Comment:
Although the Service Layer and DB Layer logic support pagination, for the time being the HTTP API only supports setting a "result limit" via the `page_size` query parameter.

If we wanted to support pagination in the future we could easily add a new optional `page_idx` parameter here.